### ISSUE TITLE
Tracking issue: Loss of lock in every Channel

### ISSUE THREAD

**Issue by matteocutugno** [gnss-sdr/gnss-sdr#192]
Hello,
I have a problem with bladerf signal. I've downloaded it from Taroz's repository (http://www.taroz.net/gnsssdrlib/data/bladerf_l1.zip) 
and i tried to process it in postprocessing.
The issue is that th tracking block doesn't work.
Here is my .conf file:

[GNSS-SDR]

;######### GLOBAL OPTIONS ##################
GNSS-SDR.internal_fs_sps=20000000

;######### SIGNAL_SOURCE CONFIG ############
SignalSource.implementation=File_Signal_Source
SignalSource.filename=/media/matteo/Archivio/20180708-blade/bladerf_l1.bin
SignalSource.item_type=ishort
SignalSource.sampling_frequency=4000000
SignalSource.freq=1575420000
;SignalSource.gain=40
;SignalSource.rf_gain=40
;SignalSource.if_gain=30
SignalSource.samples=0

;######### SIGNAL_CONDITIONER CONFIG ############
SignalConditioner.implementation=Signal_Conditioner
DataTypeAdapter.implementation=Ishort_To_Complex
InputFilter.implementation=Pass_Through
InputFilter.item_type=gr_complex
Resampler.implementation=Direct_Resampler
Resampler.sample_freq_in=2000000
Resampler.sample_freq_out=20000000
Resampler.item_type=gr_complex

;######### CHANNELS GLOBAL CONFIG ############
Channels_1C.count=8
Channels.in_acquisition=1
Channel.signal=1C

;######### ACQUISITION GLOBAL CONFIG ############
Acquisition_1C.dump=true
Acquisition_1C.dump_filename=./acquisition_ch_
Acquisition_1C.implementation=GPS_L1_CA_PCPS_Acquisition
Acquisition_1C.item_type=gr_complex
Acquisition_1C.if=0
Acquisition_1C.threshold=0.005
Acquisition_1C.sampled_ms=1
Acquisition_1C.doppler_max=10000
Acquisition_1C.doppler_min=-10000
Acquisition_1C.doppler_step=250
;Acquisition_1C.max_dwells=5

;######### TRACKING GLOBAL CONFIG ############
Tracking_1C.implementation=GPS_L1_CA_DLL_PLL_Tracking
Tracking_1C.item_type=gr_complex
Tracking_1C.if=0
Tracking_1C.dump=true
Tracking_1C.dump_filename=./tracking_ch_
Tracking_1C.pll_bw_hz=70.0;
Tracking_1C.dll_bw_hz=7.0;
Tracking_1C.order=3;

;######### TELEMETRY DECODER GPS CONFIG ############
TelemetryDecoder_1C.implementation=GPS_L1_CA_Telemetry_Decoder
TelemetryDecoder_1C.dump=true
TelemetryDecoder_1C.dump_filename=./telemetry_ch_

;######### OBSERVABLES CONFIG ############
Observables.implementation=Hybrid_Observables
Observables.dump=true
Observables.dump_filename=./obs_

;######### PVT CONFIG ############
PVT.implementation=RTKLIB_PVT
PVT.averaging_depth=100
PVT.flag_averaging=true
PVT.output_rate_ms=10
PVT.display_rate_ms=500
PVT.dump=true
PVT.dump_filename=./PVT_

I also rebuild from source gnss-sdr in debug mode to get more information in the Log file posted below:

Log file created at: 2018/07/08 17:30:54
Running on machine: matteo-VirtualBox
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
I0708 17:30:54.721213 13288 gnss_block_factory.cc:184] Getting SignalSource with implementation File_Signal_Source
I0708 17:30:54.733283 13288 gnss_block_factory.cc:230] Getting SignalConditioner with DataTypeAdapter implementation: Ishort_To_Complex, InputFilter implementation: Pass_Through, and Resampler implementation: Direct_Resampler
I0708 17:30:54.733525 13288 gnss_block_factory.cc:262] Getting Observables with implementation Hybrid_Observables
I0708 17:30:54.733757 13288 hybrid_observables_cc.cc:81] Observables dump enabled Log file: ./obs_
I0708 17:30:54.733774 13288 gnss_block_factory.cc:285] Getting PVT with implementation RTKLIB_PVT
I0708 17:30:54.734953 13288 rtklib_solver.cc:85] PVT lib dump enabled Log file: ./PVT__ls_pvt.dat
I0708 17:30:54.734982 13288 rtklib_pvt_cc.cc:380] PVT dump enabled Log file: ./PVT__raw.dat
I0708 17:30:54.735023 13288 gnss_block_factory.cc:802] Getting 8 GPS L1 C/A channels
I0708 17:30:54.735045 13288 gnss_block_factory.cc:305] Instantiating Channel 0 with Acquisition Implementation: GPS_L1_CA_PCPS_Acquisition, Tracking Implementation: GPS_L1_CA_DLL_PLL_Tracking, Telemetry Decoder implementation: GPS_L1_CA_Telemetry_Decoder
I0708 17:30:54.751539 13288 dll_pll_veml_tracking.cc:1171] Tracking Channel set to 0
I0708 17:30:54.751696 13288 dll_pll_veml_tracking.cc:1183] Tracking dump enabled on channel 0 Log file: ./tracking_ch_0.dat
I0708 17:30:54.751749 13288 gps_l1_ca_telemetry_decoder_cc.cc:177] Telemetry decoder dump enabled on channel 0 Log file: telemetry0.dat
I0708 17:30:54.794998 13288 gnss_block_factory.cc:305] Instantiating Channel 1 with Acquisition Implementation: GPS_L1_CA_PCPS_Acquisition, Tracking Implementation: GPS_L1_CA_DLL_PLL_Tracking, Telemetry Decoder implementation: GPS_L1_CA_Telemetry_Decoder
I0708 17:30:54.809641 13288 dll_pll_veml_tracking.cc:1171] Tracking Channel set to 1
I0708 17:30:54.809717 13288 dll_pll_veml_tracking.cc:1183] Tracking dump enabled on channel 1 Log file: ./tracking_ch_1.dat
I0708 17:30:54.809741 13288 gps_l1_ca_telemetry_decoder_cc.cc:177] Telemetry decoder dump enabled on channel 1 Log file: telemetry1.dat
I0708 17:30:54.828033 13288 gnss_block_factory.cc:305] Instantiating Channel 2 with Acquisition Implementation: GPS_L1_CA_PCPS_Acquisition, Tracking Implementation: GPS_L1_CA_DLL_PLL_Tracking, Telemetry Decoder implementation: GPS_L1_CA_Telemetry_Decoder
I0708 17:30:54.833274 13288 dll_pll_veml_tracking.cc:1171] Tracking Channel set to 2
I0708 17:30:54.834174 13288 dll_pll_veml_tracking.cc:1183] Tracking dump enabled on channel 2 Log file: ./tracking_ch_2.dat
I0708 17:30:54.834807 13288 gps_l1_ca_telemetry_decoder_cc.cc:177] Telemetry decoder dump enabled on channel 2 Log file: telemetry2.dat
I0708 17:30:54.853579 13288 gnss_block_factory.cc:305] Instantiating Channel 3 with Acquisition Implementation: GPS_L1_CA_PCPS_Acquisition, Tracking Implementation: GPS_L1_CA_DLL_PLL_Tracking, Telemetry Decoder implementation: GPS_L1_CA_Telemetry_Decoder
I0708 17:30:54.858515 13288 dll_pll_veml_tracking.cc:1171] Tracking Channel set to 3
I0708 17:30:54.859339 13288 dll_pll_veml_tracking.cc:1183] Tracking dump enabled on channel 3 Log file: ./tracking_ch_3.dat
I0708 17:30:54.859954 13288 gps_l1_ca_telemetry_decoder_cc.cc:177] Telemetry decoder dump enabled on channel 3 Log file: telemetry3.dat
I0708 17:30:54.875421 13288 gnss_block_factory.cc:305] Instantiating Channel 4 with Acquisition Implementation: GPS_L1_CA_PCPS_Acquisition, Tracking Implementation: GPS_L1_CA_DLL_PLL_Tracking, Telemetry Decoder implementation: GPS_L1_CA_Telemetry_Decoder
I0708 17:30:54.879552 13288 dll_pll_veml_tracking.cc:1171] Tracking Channel set to 4
I0708 17:30:54.880334 13288 dll_pll_veml_tracking.cc:1183] Tracking dump enabled on channel 4 Log file: ./tracking_ch_4.dat
I0708 17:30:54.880388 13288 gps_l1_ca_telemetry_decoder_cc.cc:177] Telemetry decoder dump enabled on channel 4 Log file: telemetry4.dat
I0708 17:30:54.896562 13288 gnss_block_factory.cc:305] Instantiating Channel 5 with Acquisition Implementation: GPS_L1_CA_PCPS_Acquisition, Tracking Implementation: GPS_L1_CA_DLL_PLL_Tracking, Telemetry Decoder implementation: GPS_L1_CA_Telemetry_Decoder
I0708 17:30:54.900768 13288 dll_pll_veml_tracking.cc:1171] Tracking Channel set to 5
I0708 17:30:54.901558 13288 dll_pll_veml_tracking.cc:1183] Tracking dump enabled on channel 5 Log file: ./tracking_ch_5.dat
I0708 17:30:54.901620 13288 gps_l1_ca_telemetry_decoder_cc.cc:177] Telemetry decoder dump enabled on channel 5 Log file: telemetry5.dat
I0708 17:30:54.918107 13288 gnss_block_factory.cc:305] Instantiating Channel 6 with Acquisition Implementation: GPS_L1_CA_PCPS_Acquisition, Tracking Implementation: GPS_L1_CA_DLL_PLL_Tracking, Telemetry Decoder implementation: GPS_L1_CA_Telemetry_Decoder
I0708 17:30:54.933760 13288 dll_pll_veml_tracking.cc:1171] Tracking Channel set to 6
I0708 17:30:54.934804 13288 dll_pll_veml_tracking.cc:1183] Tracking dump enabled on channel 6 Log file: ./tracking_ch_6.dat
I0708 17:30:54.934916 13288 gps_l1_ca_telemetry_decoder_cc.cc:177] Telemetry decoder dump enabled on channel 6 Log file: telemetry6.dat
I0708 17:30:54.969971 13288 gnss_block_factory.cc:305] Instantiating Channel 7 with Acquisition Implementation: GPS_L1_CA_PCPS_Acquisition, Tracking Implementation: GPS_L1_CA_DLL_PLL_Tracking, Telemetry Decoder implementation: GPS_L1_CA_Telemetry_Decoder
I0708 17:30:54.977332 13288 dll_pll_veml_tracking.cc:1171] Tracking Channel set to 7
I0708 17:30:54.977685 13288 dll_pll_veml_tracking.cc:1183] Tracking dump enabled on channel 7 Log file: ./tracking_ch_7.dat
I0708 17:30:54.977742 13288 gps_l1_ca_telemetry_decoder_cc.cc:177] Telemetry decoder dump enabled on channel 7 Log file: telemetry7.dat
I0708 17:30:54.997951 13288 gnss_block_factory.cc:832] Getting 0 GPS L2C (M) channels
I0708 17:30:54.997973 13288 gnss_block_factory.cc:861] Getting 0 GPS L5 channels
I0708 17:30:54.997979 13288 gnss_block_factory.cc:890] Getting 0 GALILEO E1 B (I/NAV OS) channels
I0708 17:30:54.997985 13288 gnss_block_factory.cc:919] Getting 0 GALILEO E5a I (F/NAV OS) channels
I0708 17:30:54.997990 13288 gnss_block_factory.cc:948] Getting 0 GLONASS L1 C/A channels
I0708 17:30:54.997995 13288 gnss_block_factory.cc:978] Getting 0 GLONASS L2 C/A channels
I0708 17:30:54.998245 13288 gnss_flowgraph.cc:107] Connecting flowgraph
I0708 17:30:54.998523 13288 gnss_flowgraph.cc:227] sig_source_.at(i)->get_right_block()->output_signature()->max_streams()=1
I0708 17:30:54.998531 13288 gnss_flowgraph.cc:228] sig_conditioner_.at(signal_conditioner_ID)->get_left_block()->input_signature()=1
I0708 17:30:54.998534 13288 gnss_flowgraph.cc:240] connecting sig_source_ 0 stream 0 to conditioner 0
I0708 17:30:55.002918 13288 gnss_flowgraph.cc:522] Channel 0 assigned to GPS PRN 01 (Block IIF) Signal 1C
I0708 17:30:55.002941 13288 gnss_flowgraph.cc:529] Channel 0 connected to observables and ready for acquisition
I0708 17:30:55.002946 13288 gnss_flowgraph.cc:522] Channel 1 assigned to GPS PRN 02 (Block IIR) Signal 1C
I0708 17:30:55.002951 13288 gnss_flowgraph.cc:533] Channel 1 connected to observables in standby mode
I0708 17:30:55.002955 13288 gnss_flowgraph.cc:522] Channel 2 assigned to GPS PRN 03 (Block IIF) Signal 1C
I0708 17:30:55.002959 13288 gnss_flowgraph.cc:533] Channel 2 connected to observables in standby mode
I0708 17:30:55.002962 13288 gnss_flowgraph.cc:522] Channel 3 assigned to GPS PRN 04 (Block Unknown) Signal 1C
I0708 17:30:55.002966 13288 gnss_flowgraph.cc:533] Channel 3 connected to observables in standby mode
I0708 17:30:55.002969 13288 gnss_flowgraph.cc:522] Channel 4 assigned to GPS PRN 05 (Block IIR-M) Signal 1C
I0708 17:30:55.002974 13288 gnss_flowgraph.cc:533] Channel 4 connected to observables in standby mode
I0708 17:30:55.002977 13288 gnss_flowgraph.cc:522] Channel 5 assigned to GPS PRN 06 (Block IIF) Signal 1C
I0708 17:30:55.002980 13288 gnss_flowgraph.cc:533] Channel 5 connected to observables in standby mode
I0708 17:30:55.002984 13288 gnss_flowgraph.cc:522] Channel 6 assigned to GPS PRN 07 (Block IIR-M) Signal 1C
I0708 17:30:55.002987 13288 gnss_flowgraph.cc:533] Channel 6 connected to observables in standby mode
I0708 17:30:55.002990 13288 gnss_flowgraph.cc:522] Channel 7 assigned to GPS PRN 08 (Block IIF) Signal 1C
I0708 17:30:55.002993 13288 gnss_flowgraph.cc:533] Channel 7 connected to observables in standby mode
I0708 17:30:55.003065 13288 gnss_flowgraph.cc:538] Flowgraph connected
I0708 17:30:55.003082 13288 control_thread.cc:124] Flowgraph connected
I0708 17:30:55.010291 13288 control_thread.cc:135] Flowgraph started
I0708 17:30:55.118157 13305 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 01 (Block IIF) on channel 0
I0708 17:30:55.118175 13305 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = -7500. Code Phase correction [samples] = 1.53831. PULL-IN Code Phase [samples] = 9598.46
I0708 17:30:55.118340 13288 gnss_flowgraph.cc:898] Channel 0 ACQ SUCCESS satellite GPS PRN 01 (Block IIF)
I0708 17:30:55.676125 13303 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 10 (Block IIF) on channel 2
I0708 17:30:55.676151 13303 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = 3750. Code Phase correction [samples] = -0.302661. PULL-IN Code Phase [samples] = 10652.3
I0708 17:30:55.676247 13288 gnss_flowgraph.cc:898] Channel 2 ACQ SUCCESS satellite GPS PRN 10 (Block IIF)
I0708 17:30:55.741979 13304 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 11 (Block IIR) on channel 1
I0708 17:30:55.742400 13304 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = 4750. Code Phase correction [samples] = -0.632662. PULL-IN Code Phase [samples] = 8584.63
I0708 17:30:55.743997 13288 gnss_flowgraph.cc:898] Channel 1 ACQ SUCCESS satellite GPS PRN 11 (Block IIR)
I0708 17:30:55.846561 13301 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 13 (Block IIR) on channel 4
I0708 17:30:55.846577 13301 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = 6500. Code Phase correction [samples] = -0.284437. PULL-IN Code Phase [samples] = 13191.3
I0708 17:30:55.846729 13288 gnss_flowgraph.cc:898] Channel 4 ACQ SUCCESS satellite GPS PRN 13 (Block IIR)
I0708 17:30:55.974824 13302 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 14 (Block IIR) on channel 3
I0708 17:30:55.974953 13302 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = 1750. Code Phase correction [samples] = -0.30545. PULL-IN Code Phase [samples] = 14020.3
I0708 17:30:55.975041 13288 gnss_flowgraph.cc:898] Channel 3 ACQ SUCCESS satellite GPS PRN 14 (Block IIR)
I0708 17:30:56.103157 13299 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 16 (Block IIR) on channel 6
I0708 17:30:56.103176 13299 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = 1500. Code Phase correction [samples] = -0.0938368. PULL-IN Code Phase [samples] = 1753.09
I0708 17:30:56.103235 13288 gnss_flowgraph.cc:898] Channel 6 ACQ SUCCESS satellite GPS PRN 16 (Block IIR)
I0708 17:30:56.249469 13298 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 18 (Block IIR) on channel 7
I0708 17:30:56.249491 13298 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = -3250. Code Phase correction [samples] = 0.210691. PULL-IN Code Phase [samples] = 6946.79
I0708 17:30:56.249614 13288 gnss_flowgraph.cc:898] Channel 7 ACQ SUCCESS satellite GPS PRN 18 (Block IIR)
I0708 17:30:56.403321 13300 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 20 (Block IIR) on channel 5
I0708 17:30:56.403338 13300 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = -1500. Code Phase correction [samples] = 0.0613504. PULL-IN Code Phase [samples] = 15220.9
I0708 17:30:56.403398 13288 gnss_flowgraph.cc:898] Channel 5 ACQ SUCCESS satellite GPS PRN 20 (Block IIR)
I0708 17:31:02.241276 13338 dll_pll_veml_tracking.cc:650] Loss of lock in channel 2!
I0708 17:31:02.241569 13288 gnss_flowgraph.cc:928] Channel 2 TRK FAILED satellite GPS PRN 10 (Block IIF)
I0708 17:31:02.241590 13288 gnss_flowgraph.cc:935] Channel 2 Starting acquisition GPS PRN 10 (Block IIF), Signal 1C
I0708 17:31:02.323979 13303 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 10 (Block IIF) on channel 2
I0708 17:31:02.327455 13303 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = 3750. Code Phase correction [samples] = -0.411456. PULL-IN Code Phase [samples] = 10643.4
I0708 17:31:02.327828 13288 gnss_flowgraph.cc:898] Channel 2 ACQ SUCCESS satellite GPS PRN 10 (Block IIF)
I0708 17:31:02.507894 13333 dll_pll_veml_tracking.cc:650] Loss of lock in channel 3!
I0708 17:31:02.508146 13288 gnss_flowgraph.cc:928] Channel 3 TRK FAILED satellite GPS PRN 14 (Block IIR)
I0708 17:31:02.508193 13288 gnss_flowgraph.cc:935] Channel 3 Starting acquisition GPS PRN 14 (Block IIR), Signal 1C
I0708 17:31:02.554081 13302 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 14 (Block IIR) on channel 3
I0708 17:31:02.554097 13302 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = 1750. Code Phase correction [samples] = -0.0609748. PULL-IN Code Phase [samples] = 10623.1
I0708 17:31:02.554296 13288 gnss_flowgraph.cc:898] Channel 3 ACQ SUCCESS satellite GPS PRN 14 (Block IIR)
I0708 17:31:02.729274 13313 dll_pll_veml_tracking.cc:650] Loss of lock in channel 7!
I0708 17:31:02.729550 13288 gnss_flowgraph.cc:928] Channel 7 TRK FAILED satellite GPS PRN 18 (Block IIR)
I0708 17:31:02.729569 13288 gnss_flowgraph.cc:935] Channel 7 Starting acquisition GPS PRN 18 (Block IIR), Signal 1C
I0708 17:31:02.807626 13298 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 18 (Block IIR) on channel 7
I0708 17:31:02.807643 13298 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = 3250. Code Phase correction [samples] = -0.391284. PULL-IN Code Phase [samples] = 7856.39
I0708 17:31:02.807739 13288 gnss_flowgraph.cc:898] Channel 7 ACQ SUCCESS satellite GPS PRN 18 (Block IIR)
I0708 17:31:03.398886 13323 dll_pll_veml_tracking.cc:650] Loss of lock in channel 5!
I0708 17:31:03.399262 13288 gnss_flowgraph.cc:928] Channel 5 TRK FAILED satellite GPS PRN 20 (Block IIR)
I0708 17:31:03.399291 13288 gnss_flowgraph.cc:935] Channel 5 Starting acquisition GPS PRN 20 (Block IIR), Signal 1C
I0708 17:31:03.461356 13300 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 20 (Block IIR) on channel 5
I0708 17:31:03.461380 13300 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = -1500. Code Phase correction [samples] = 0.0538695. PULL-IN Code Phase [samples] = 15192.9
I0708 17:31:03.461503 13288 gnss_flowgraph.cc:898] Channel 5 ACQ SUCCESS satellite GPS PRN 20 (Block IIR)
I0708 17:31:03.684963 13348 dll_pll_veml_tracking.cc:650] Loss of lock in channel 0!
I0708 17:31:03.685575 13288 gnss_flowgraph.cc:928] Channel 0 TRK FAILED satellite GPS PRN 01 (Block IIF)
I0708 17:31:03.685600 13288 gnss_flowgraph.cc:935] Channel 0 Starting acquisition GPS PRN 01 (Block IIF), Signal 1C
I0708 17:31:03.784517 13305 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 01 (Block IIF) on channel 0
I0708 17:31:03.784535 13305 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = -7500. Code Phase correction [samples] = 0.433548. PULL-IN Code Phase [samples] = 5490.57
I0708 17:31:03.784607 13288 gnss_flowgraph.cc:898] Channel 0 ACQ SUCCESS satellite GPS PRN 01 (Block IIF)
I0708 17:31:04.373543 13318 dll_pll_veml_tracking.cc:650] Loss of lock in channel 6!
I0708 17:31:04.373654 13288 gnss_flowgraph.cc:928] Channel 6 TRK FAILED satellite GPS PRN 16 (Block IIR)
I0708 17:31:04.373677 13288 gnss_flowgraph.cc:935] Channel 6 Starting acquisition GPS PRN 16 (Block IIR), Signal 1C
I0708 17:31:04.594424 13299 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 16 (Block IIR) on channel 6
I0708 17:31:04.594440 13299 dll_pll_veml_tracking.cc:527] PULL-IN Doppler [Hz] = -7500. Code Phase correction [samples] = 1.55087. PULL-IN Code Phase [samples] = 12986.4
I0708 17:31:04.594501 13288 gnss_flowgraph.cc:898] Channel 6 ACQ SUCCESS satellite GPS PRN 16 (Block IIR)
I0708 17:31:09.189141 13338 dll_pll_veml_tracking.cc:650] Loss of lock in channel 2!
I0708 17:31:09.189479 13288 gnss_flowgraph.cc:928] Channel 2 TRK FAILED satellite GPS PRN 10 (Block IIF)
I0708 17:31:09.189508 13288 gnss_flowgraph.cc:935] Channel 2 Starting acquisition GPS PRN 10 (Block IIF), Signal 1C
I0708 17:31:09.318686 13303 dll_pll_veml_tracking.cc:519] Starting tracking of satellite GPS PRN 10 (Block IIF) on channel 2
....... and so on

It seems that DLL and PLL loops can't estimate doppler shift or something else. I tried with different parameters in acquisition block( Doppler.max, Doppler.step and Doppler.Threshold)
and also in Tracking block (bandwidth of dll and pll)

Thank you in advance for your help

Matteo

**Comment by Paulth18** [comment]
Are you sure that this " SignalSource.sampling_frequency=4000000" is correct? Try with 2

And then, why you are using bw so large for filter?
Tracking_1C.pll_bw_hz=70.0;
Tracking_1C.dll_bw_hz=7.0;
Try with more narrow value.

Anyway try with some other file, if the problem remain comment here again.

**Comment by matteocutugno** [comment]
Thank you for your answer.

I tried with SignalSource.sampling_frequency=2000000
And I also tried to narrowing bw.
It continues to loss lock.
With other signals like Spain sample (my first position fix) it works.
I was interested in testing bladerf x40 signal sample because i've just ordered one and next week i hope they will deliver to me. Soi'm afraid i'm not able to make it work.

Anyway i dont't understand how i have to set some paremeters of acquisition and tracking loop that depends on the signal like:
Acquisition_1C.threshold
Acquisition_1C.doppler_max
Acquisition_1C.doppler_min
Acquisition_1C.doppler_step
Tracking_1C.pll_bw_hz
Tracking_1C.dll_bw_hz

Thank you

Matteo
